garrysmod/gamemodes/sandbox/gamemode/spawnmenu/controlpanel.lua

```diff
@@ -91,6 +91,27 @@ function PANEL:KeyBinder( label1, convar1, label2, convar2 )
 
 end
 
+function PANEL:ColorPicker( label, convarr, convarg, convarb, convara )
+
+	local color = vgui.Create( "CtrlColor", self )
+
+	color:Dock( TOP )
+	color:SetLabel( label )
+
+	color:SetConVarR( convar )

```

Comment:
BUG: `convar` doesn't exist. Probably should name these something like `convarR`, `convarG`, etc so its easier to see.

Reply:
Fixed: https://github.com/Facepunch/garrysmod/pull/1854/commits/e57c7cbf70399d62161f5e185aabc2397ee693d2